frontend/src/pages/Assessments.js

```diff
@@ -1,13 +1,98 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
 import Layout from "../components/Layout";
-import PromptSpeech from "../components/PromptSpeech";
+import { Button } from "antd";
+import { INK } from "../constants/colors";
+
+const landingImageUrl =
+  "https://firebasestorage.googleapis.com/v0/b/vocab-buddy-53eca.appspot.com/o/assessments-landing.png?alt=media&token=168586ea-bb4e-499c-a711-5f3fe7a375b9";
+
+const AssessmentsContainer = styled.div`
+  display: flex;
+  font-size: 56px;
+  font-weight: bold;
+  margin-top: 60px;
+`;
+
+const AssessmentsContainerMobile = styled.div`
+  font-size: 56px;
+  font-weight: bold;
+  width: 100%;
+`;
+
+const Prompt = styled.div`
+  font-size: 20px;
+  font-weight: normal;
+  margin-top: 20px;
+`;
+
+const LandingPageImage = styled.img`
+  flex: 1;
+  margin-top: 30px;
+  width: 100%;
+`;
+
+const LandingPageText = styled.div`
+  flex: 1;
+  margin-left: 1em;
+  margin-top: 120px;
+`;
+
+const BeginButtonMobile = styled(Button)`
+  background: ${INK};
+  border: none;
+  border-radius: 12px;
+  font-weight: bold;
+  height: 50px;
+
+  :hover {
+    background: red;
+  }
+`;
+
+const BeginButton = styled(BeginButtonMobile)`
+  margin-top: 100px;
+  width: 120px;
+`;
 
 const Assessments = () => {
+  const [screenWidth, setScreenWidth] = useState(window.innerWidth);
+
+  useEffect(() => {
+    const resizeScreen = () => {
+      setScreenWidth(window.innerWidth);
+    };
+    window.addEventListener("resize", resizeScreen);
+    return () => {
+      window.removeEventListener("resize", resizeScreen);
+    };
+  }, []);
```

Comment:
@cindy1u0 I wonder if we want to do screen size like this, or use CSS to do it with an `@media` tag. I know in my pr I used the `@media` tag which in my opinion looks cleaner, but in your ppt you used `state` with width. What do you think?

Reply:
Oh ya I think for this, might be better to put it in the css component. In my PR, I had resize because there are some components we need to show or hide when the screen size is smaller than 600. I think you can make the container display: flex, then change the flex direction depending on how big the screens size is.
```
@media (max-width: 600px)`
   flex-direction: column;
`
```